scripts/figma-to-css.mjs

```diff
@@ -193,6 +193,7 @@ const figmaToCss = async () => {
       && !(name.includes('space') || name.includes('focus'))
       && getVariableHead(name) !== getVariableHead(tokens[index - 1][0])
       ? '\n' : '';
+      if (name.includes('letter-spacing')) value = value.split('\n')[1];
```

Comment:
I think we should do that in a different way. 
Check this part of `getCSSVars` args
```
setValue = (value) => value.replace(/\n.+/g, '')
```
You can use a custom `setValue` function 
<img src="https://media1.giphy.com/media/6UFgdU9hirj1pAOJyN/giphy.gif"/>

Go to line 222
you can solve this issue in one line ;) with array functions
1. Transform string to array
2. Filter from array value with not allowed units
3. Transform the array with a single element to sting  
4. Return 🦄